docs/framework/react/guide/virtual-file-routes.md

```diff
@@ -46,15 +47,34 @@ Virtual file routes can be configured either via:
 
 ## Configuration via the TanStackRouter Plugin
 
-If you're using the `TanStackRouter` plugin for Vite/Rspack/Webpack, you can configure virtual file routes by passing a `virtualRoutesConfig` option to the plugin:
+If you're using the `TanStackRouter` plugin for Vite/Rspack/Webpack, you can configure virtual file routes by passing the path of your routes file to the `virtualRoutesConfig` option when setting up the plugin:
```

Comment:
since we can still pass `virtualRoutesConfig` as an object into the config, please describe both ways